controllers/azurecluster_controller.go

```diff
@@ -267,10 +267,6 @@ func (acr *AzureClusterReconciler) reconcileDelete(ctx context.Context, clusterS
 	log.Info("Reconciling AzureCluster delete")
 
 	azureCluster := clusterScope.AzureCluster
-	conditions.MarkFalse(azureCluster, infrav1.NetworkInfrastructureReadyCondition, clusterv1.DeletedReason, clusterv1.ConditionSeverityInfo, "")
```

Comment:
this condition was useful before we had conditions for each resource type but now it's just overshadowing other conditions that have the same severity in the "Ready" summary and it happened to be shown first because of lexicographical order